src/components/common/ContactCustomerSupportButton.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Button } from '@edx/paragon';
 import PropTypes from 'prop-types';
 
-import { configuration } from '../../../config';
+import { configuration } from '../../config';
 
 const ContactCustomerSupportButton = (props) => (
```

Comment:
Moved this component to `/src/components/common/ContactCustomerSupportButton.jsx` so that it can be more readably accessed.

Reply:
no action here needed but should we just take it out of common? There's no other components in there.